core/templates/dev/head/forms/formBuilder.js

```diff
@@ -1163,7 +1163,10 @@ oppia.directive('schemaBasedEditor', [function() {
       onInputFocus: '='
     },
     templateUrl: 'schemaBasedEditor/master',
-    restrict: 'E'
+    restrict: 'E',
+    controller: function() {
+      document.execCommand('enableObjectResizing', false, false);
```

Comment:
Does it work if you do it instead in the textAngular provider? (Grep for taOptions)